Code/Tools/TestImpactFramework/Runtime/Code/Include/TestImpactFramework/TestImpactClientSequenceReport.h

```diff
@@ -368,7 +368,7 @@ namespace TestImpact
                 const AZStd::vector<AZStd::string>& draftedTestRuns,
                 TestRunReport&& selectedTestRunReport,
                 TestRunReport&& draftedTestRunReport)
-                : SequenceReportBase(
```

Comment:
Do you need the explicit template parameter here? I'd expect it could deduce it from the `policyState` argument.

Reply:
it complain without it